src/components/views/project/projectDonations/QfRoundSelector.tsx

```diff
@@ -37,9 +37,18 @@ export const QfRoundSelector: FC<IQfRoundSelectorProps> = ({
 	const navigationNextRef = useRef(null);
 
 	const sortedRounds =
-		projectData?.qfRounds?.sort(
-			(a, b) => Number(b.isActive) - Number(a.isActive),
-		) || [];
+		projectData?.qfRounds?.sort((a, b) => {
+			// First, compare by isActive
+			const isActiveComparison = Number(b.isActive) - Number(a.isActive);
+
+			// If isActive values are the same, compare by name
+			if (isActiveComparison === 0) {
+				return a.name.localeCompare(b.name);
+			}
+
+			// Otherwise, return the comparison result of isActive
+			return isActiveComparison;
+		}) || [];
```

Comment:
this would first sort rounds based on isActive prop and then sort all active and non active rounds